core/test/integration/api/api_posts_spec.js

```diff
@@ -594,6 +594,34 @@ describe('Post API', function () {
             });
         });
 
+        it('can fetch post returning a slug only permalink', function () {
```

Comment:
> browse posts: request to include tags with absolute_urls

I think it's worth it to extend this test in core/test/functional/routes/api/public_api_spec.js.

Include everything you can and check all urls. Same for without absolute_urls=true. Only for the public api.

Reply:
Added more even more test coverage to public_api_spec.js